src/test/kotlin/no/nav/dagpenger/journalføring/arena/DokumentAdapterTest.kt

```diff
@@ -0,0 +1,29 @@
+package no.nav.dagpenger.journalføring.arena
+
+import com.squareup.moshi.Types
+import io.kotlintest.shouldBe
+import io.kotlintest.shouldNotBe
+import no.nav.dagpenger.events.moshiInstance
+import org.junit.jupiter.api.Test
+
+class DokumentAdapterTest {
+    data class DokumentInfo(val tittel: String, val brevkode: String)
+    @Test
+    fun `Skal greie å konvertere dokumentInfo med flere felter til Dokument med tittel`() {
+        val dokumenter = listOf(DokumentInfo("en", "123"), DokumentInfo("to", "123"))
```

Comment:
Kan muligens diskuteres om det er tydeligere aa teste mot en json string istedet for dataklasse og Moshi adapter.

Reply:
enig der altså